rdmd_test.d

```diff
@@ -88,65 +91,65 @@ void runTests()
     string forceSrc = tempDir().buildPath("force_src_.d");
     std.file.write(forceSrc, `void main() { pragma(msg, "compile_force_src"); }`);
 
-    res = execute([rdmdApp, compilerSwitch, forceSrc]);
+    res = execute([rdmdApp, compilerSwitch, modelSwitch, forceSrc]);
```

Comment:
> Is there any instance of compilerSwitch that's not accompanied by an instance of of modelSwitch?

Nope, but as `execute` excepts an array, we can't do `["dmd", "--compiler=foo -m32", ...]`.
Btw (a bit OT): is there any reason that `execute` doesn't allow ranges?